SPECS/xcp-clipboardd.spec

```diff
@@ -0,0 +1,36 @@
+Name:           xcp-clipboardd
+Version:        1.0.2
+Release:        1
+Summary:        Daemon to share a virtualized Windows clipboard
+License:        GPLv3
+URL:            https://github.com/xcp-ng/xcp-clipboardd
+Source0:        https://github.com/xcp-ng/xcp-clipboardd/archive/v%{version}/%{name}-%{version}.tar.gz
+
+BuildRequires:  cmake3
+BuildRequires:  make
+BuildRequires:  gcc
+BuildRequires:  xen-libs-devel
+
+Obsoletes:      xs-clipboardd
+
+%description
+Share clipboard between guest Windows and host with VNC.
+
+%prep
+%autosetup -p1
+
+%build
+mkdir build && cd build
+cmake3 .. -DCMAKE_INSTALL_PREFIX=%{buildroot}/opt/xensource
+make
+
+%install
+cd build
+make install
+
+%files
+/opt/xensource/libexec/xcp-clipboardd
```

Comment:
We should also provide documentation, at least LICENSE and maybe README.md.

`%doc LICENSE README.md`

Reply:
`%doc` usually goes right after `%files`: https://src.fedoraproject.org/cgit/rpms/nodejs-domhandler.git/tree/nodejs-domhandler.spec#n53

And we should either provide a better README.md or remove it from package docs

Reply:
@benjamreis Have you finished your documentation of this binary? Can I add it in this file?

Reply:
@Wescoeur yep it's finished. If everyone's okay with it it can be added :)